src/components/Typography/HX/HX.tsx

```diff
@@ -0,0 +1,62 @@
+import { ReactElement, ReactNode } from 'react'
+import { Typography as AntTypography } from 'antd'
+
+import { CopyConfig, EllipsisConfig } from '../Typography.types'
+
+const { Title: AntTitle } = AntTypography
+
+export type HXProps = {
+
```

Comment:
We decided not to uese these new lines

Reply:
We decided not to have a new line between a function declaration and its implementation, but here we are documenting the properties of the component and I think the new line slightly improves readability since it will always be followed by a comment